tests/Query/Filter/DueDateField.test.ts

```diff
@@ -409,29 +468,42 @@ describe('due date after relative date range (Today is 2021-11-01)', () => {
         ['this year', '2020-12-31', '2021-01-01', '2021-12-31', '2022-01-01'],
         ['next year', '2021-12-31', '2022-01-01', '2022-12-31', '2023-01-01'],
     ])(
-        'due after %s: task with due date %s, %s, %s are not included, %s is',
+        'due after & in or after %s',
         (range: string, beforeRange: string, rangeStart: string, rangeEnd: string, afterRange: string) => {
             // Arrange
-            const filter = new DueDateField().createFilterOrErrorMessage(`due after ${range}`);
+            const filterAfter = new DueDateField().createFilterOrErrorMessage(`due after ${range}`);
+            const filterOnOrAfter = new DueDateField().createFilterOrErrorMessage(`due in or after ${range}`);
```

Comment:
These are more tests that very much depend on believing the data supplied in the array... And then having to figure out the logic of the test values.

I think that this could benefit from being converted to an approval test, where the output file is formatted to make the patterns in the filters and the explanations clear.